kernel/enter.c

```diff
@@ -657,21 +651,26 @@ int myst_enter_kernel(myst_kernel_args_t* args)
     /* myst_handle_host_signal can be called from enclave exception handlers */
     args->myst_handle_host_signal = myst_handle_host_signal;
 
-    /* Save the aguments */
+    /* make a copy of the input kernel aguments and reassign args */
     __myst_kernel_args = *args;
+    args = &__myst_kernel_args;
+
+    /* turn off various options when TEE is not in debug mode */
+    if (!args->tee_debug_mode)
+    {
+        args->trace_errors = false;
```

Comment:
`myst_eprintf()` used in a lot of code, but it doesn't seem to be under `trace_errors` control.

Reply:
This is intentional since users still need to know about certain error conditions in release mode (e.g., failure to mount a file system because the root hash is wrong). I did a pass over all the calls to ``myst_eprintf`` I cannot see any case where secrets would be leak (except for the cases that are already disabled by various trace options).

Reply:
Saw the latest change to guard backtrace with tee_debug_mode flag. All other existing myst_eprintf() usages don't expose any info about the app.